File: 7KYU/Sum of the first nth term of Series/solution.clj
(ns nthseries.core)

(defn series-sum [n]
  (loop [total 0
         c 1
         d 1]
    (if (> c n)
      (format "%.2f"(double total))
      (recur (+ total (/ 1.0 d)) (inc c) (+ 3 d)))))

File: 7KYU/Sum of the first nth term of Series/solution.cs
using System;

public class NthSeries
{
	public static string seriesSum (int n)
	{
		if(n == 0)
			return "0.00";
      
		double start = 4;
		double total = 1;
		for(double i = 1; i < n; i++)
		{
			total += (1 / (double)start);
			start += 3;
		}
		return total.ToString(".00");
	}
}